ospf6d/ospf6d.h

```diff
@@ -21,6 +21,7 @@
 #ifndef OSPF6D_H
 #define OSPF6D_H
 
+#include "config.h"
```

Comment:
`#include "config.h"` does not belong in `.h` files, instead the first include in all `.c` files must be either `config.h` or `zebra.h`.

Reply:
It was added for some debug. Removed it.